lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -155,6 +155,8 @@ sub cache_assets {
         }
         if (!$asset) {
             $error = "Failed to download $asset_uri to " . $cache_client->asset_path($webui_host, $asset_uri);
+            $error .= '. Asset was pruned immediately after download (poo#71827), please retrigger'
+              if $msg =~ /Purging.*$asset_uri/;
```

Comment:
I would add a comment like `# note: This check does not work if the download was performed by an already enqueued Minion job or if the pruning happened within a completely different asset download.`.

Reply:
I have included a slightly rephrased variant, i.e. `# note: This check has no effect if …` instead of "does not work"